packages/protocol/contracts/common/UsingFixidity.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.8;
+
+import "fixidity/contracts/FixidityLib.sol";
+
+
+contract UsingFixidity {
+  using FixidityLib for int256;
+
+  int256 public constant FIXED1 = 1000000000000000000000000;
```

Comment:
Fixidity has a `function fixed1() public pure returns(int256)`

Reply:
Since this is a linked library, I think every call to that function will incur about a 2k gas fee.

Reply:
yeah, makes sense. Let's add a comment that this is the result of doing `fixed1()`